SolveSATWithGrover/Tests.qs

```diff
@@ -336,13 +323,13 @@ namespace Quantum.Kata.GroversAlgorithm {
     @Test("QuantumSimulator")
     operation T22_Oracle_Exactly1SAT_Test () : Unit {
         // General SAT instances
-        for (i in 1..10) {
-            let (nVar, problem) = Generate_SAT_Instance(3);
-            Message($"Testing exactly-1 3-SAT instance ({nVar}, {SATInstanceAsString(problem)})...");
+        for (i in 2..6) { // test for 2..6 variables
+            let problem = GenerateSATInstance(i,3,3); // 3 clauses, and each clause 3 terms
```

Comment:
Same here, I'd suggest using `nVar` for loop variable and some variance in the number of clauses
```suggestion
            let problem = GenerateSATInstance(nVar, nVar - 1,3);
```

Reply:
Addressed in latest commit